README.md

```
@@ -40,3 +40,13 @@ bun dev
 ```
 
 You can see the full list of commands in each of the `package.json` files in the `apps` and `services` directories.
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)